### DISCUSSION TITLE
Biochar model

### DISCUSSION THREAD

**Discussion by albanif** [APSIMInitiative/ApsimX#9382] [Q&A] [answered]
Hello APSIM community,
Is the biochar model (https://www.apsim.info/biochar-model/) available in NextGen? If not, is there a reason why this hasn’t been converted yet?
Thanks

**Comment by hut104** [comment, marked as answer]
Hi,
The biochar model is not included within the APSIM Next Generation release.

We take a more deliberate approach within Next Generation for the deployment of models.  To complete this task correctly, we will require test datasets as part of our continual release/ continual delivery approach.  We would then prefer to implement within the nutrient modelling framework rather than within a script.  In this way, and changes to the organic matter model (or any other model really) will automatically be tested for impacts on the biochar modelling capacity.

Your request may be the first for the addition of such functionality.  We could investigate options for implementing such a capability.

**Reply by albanif** [reply]
Hi,
that makes perfectly sense. Please let me know how I could support the work to make this happening. We would need the biochar module for a project in south Asia and another in UK. 
Many thanks
Fabrizio
